tests/integration/test_row_checks.py

```diff
@@ -1221,3 +1222,44 @@ def test_is_ipv4_address_in_cidr(spark):
         checked_schema,
     )
     assert_df_equality(actual, expected, ignore_nullable=True)
+
+
+def test_is_data_fresh(spark):
```

Comment:
Please also add this check here: https://github.com/databrickslabs/dqx/blob/main/tests/resources/all_row_checks.yaml